mix.exs

```diff
@@ -103,7 +103,7 @@ defmodule DotCom.Mixfile do
       {:jason, "1.4.4", override: true},
       {:logster, "1.1.1"},
       {:mail, "0.3.1"},
-      {:mbta_metro, "0.0.49"},
+      {:mbta_metro, "0.0.50"},
```

Comment:
Was upgrading metro necessary as part of this PR?

Reply:
No, but we might as well.

Reply:
I won't block approval on this, but in general I don't like lumping library upgrades together with unrelated features. It runs the risk of creating hard-to-revert surprises, especially when the library is in alpha, as metro is.